README.md

```diff
@@ -1,14 +1,47 @@
 # Flexbox playground
-[![All Contributors](https://img.shields.io/badge/all_contributors-4-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors)
+
+# How to run the app - Available Scripts
```

Comment:
There's already a PR adding documentation on how to run the app

Reply:
Nop.